src/pretalx/htmlexport/management/commands/export_schedule_html.py

```diff
@@ -0,0 +1,33 @@
+from bakery.management.commands.build import Command as BakeryBuildCommand
+from django.core.management import call_command
+from django.core.management.base import CommandError
+from django.core.urlresolvers import get_callable
+from django.utils import translation
+
+from pretalx.event.models import Event
+
+
+class Command(BakeryBuildCommand):
+    help = 'Exports event schedule as a static HTML dump'
+
+    def add_arguments(self, parser):
+        super().add_arguments(parser)
+        parser.add_argument('event', type=str)
+
+    def handle(self, *args, **options):
+        try:
+            event = Event.objects.get(slug=options['event'])
```

Comment:
Please use ``slug__iexact=`` instead as we do everywhere.

Reply:
gotcha. ✅